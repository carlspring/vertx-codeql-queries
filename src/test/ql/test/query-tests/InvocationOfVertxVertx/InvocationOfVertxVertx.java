package org.carlspring.security.vertx.http;

import io.vertx.core.Vertx;

/**
 * @author carlspring
 */
public class InvocationOfVertxVertx
{

    public void start()
    {
        Vertx.vertx();
    }

}
