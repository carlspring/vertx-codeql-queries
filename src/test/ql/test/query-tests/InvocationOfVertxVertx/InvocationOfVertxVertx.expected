| InvocationOfVertxVertx.java:13:9:13:21 | vertx(...) | Invocation of VertX.vertx() |
